core/src/avm2/globals/flash/events/ThrottleEvent.as

```diff
@@ -5,6 +5,7 @@
 package flash.events
 {
     
+    [API("676")]
```

Comment:
I think this should be `674` for Flash Player 11:
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/events/ThrottleEvent.html